.github/workflows/deploy.yml

```diff
@@ -37,11 +37,7 @@ jobs:
         script: |
           cd ~/penumbra
           docker-compose stop
-          rm -rf ~/scratch/testnet_build/node0/tendermint/data/
-          rm -rf ~/scratch/testnet_build/node0/pd/rocksdb
-          mkdir ~/scratch/testnet_build/node0/tendermint/data/
-          echo -n '{}' > ~/scratch/testnet_build/node0/tendermint/data/priv_validator_state.json
-          chmod -R 777 ~/scratch/testnet_build
```

Comment:
@zbuc I wasn't sure why this needed to manually construct this data, rather than relying on `generate-testnet`, but it seems to work just to just blow away the whole testnet directory.